auth0/v3/management/grants.py

```diff
@@ -0,0 +1,65 @@
+from .rest import RestClient
+
+
+class Grants(object):
+
+    """Auth0 grants endpoints
+
+    Args:
+        domain (str): Your Auth0 domain, e.g: 'username.auth0.com'
+
+        token (str): Management API v2 Token
+
+        telemetry (bool, optional): Enable or disable Telemetry
+            (defaults to True)
+    """
+
+    def __init__(self, domain, token, telemetry=True):
+        self.domain = domain
+        self.client = RestClient(jwt=token, telemetry=telemetry)
+
+    def _url(self, id=None):
+        url = 'https://%s/api/v2/grants' % self.domain
+        if id is not None:
+            return url + '/' + id
+        return url
+
+    def get_all(self, user_id=None, client_id=None, audience=None, page=None, per_page=None, include_totals=False):
+        """Retrieves all grants.
+
+        Args:
+            audience (str, optional): URL encoded audience of a Resource Server
+                to filter
+
+            page (int, optional): The result's page number (zero based).
+
+            per_page (int, optional): The amount of entries per page.
+
+            include_totals (bool, optional): True if the query summary is
```

Comment:
Probably want to mention that the data structure returned is different (object instead of an array)